src/strategies/index.ts

```diff
@@ -179,5 +181,7 @@ export default {
   apescape,
   liftkitchen,
   'decentraland-estate-size': decentralandEstateSize,
+  'xrc20-balance-of': xrc20BalanceOf,
+  'iotex-balance': iotexBalance,
```

Comment:
@maswalker If possible please add  `example.json` file for these strategies and make sure to run the `npm test --strategy=xrc20-balance-of` command to test these strategies

Reply:
test examples added, and I run the 'npm test --strategy=xrc20-balance-of' and 'npm test --strategy=iotex-balance' successfully.

please check it again, thanks. @ChaituVR @bonustrack